solr/packaging/test/test_bats.bats

```diff
@@ -21,7 +21,7 @@ load bats_helper
 
 setup_file() {
   # set up paths and helpers
-  common_setup
+  common_clean_setup
```

Comment:
Can you add a comment here about how this is different from `common_setup` (and possibly point out that we don't want to call it in teardown)?

Reply:
Either a comment here (in test_bats) since that is supposed to be our documented example, or a function on bats_helper since that is where the function is declared. Not required on each use of it.